sematic/ui/src/hooks/graphHooks.ts

```diff
@@ -43,20 +38,50 @@ export function useGraph(runRootId: string): [
 
     }, [graphPayload]);
 
+    const retryPending = useRef(false);
+
+    const prevLoading = usePrevious(loading);
+
+    useEffect(() => {
+        if (prevLoading&& !loading && retryPending.current) {
+            retryPending.current = false;
+            devLogger('Loading state has switched from true to false,' 
+                + ' and reloading was requested. reloading now...');
```

Comment:
Please use he double quote style consistently in all new code.
```suggestion
            devLogger("Loading state has switched from true to false," 
                + " and reloading was requested. Reloading now...");
```